backend/apps/menu/views.py

```
@@ -126,6 +126,105 @@ def current(self, request):
+            reader = csv.DictReader(io_string, delimiter=';')
+            if not reader.fieldnames or 'day' not in [f.strip().lower() for f in reader.fieldnames]:
+                io_string.seek(0)
+                reader = csv.DictReader(io_string, delimiter=',')
+                if not reader.fieldnames or 'day' not in [f.strip().lower() for f in reader.fieldnames]:
+                    return Response({"detail": "Formato de CSV inválido. Asegúrese de incluir las columnas requeridas."}, status=status.HTTP_400_BAD_REQUEST)
+        except Exception as e:
+            return Response({"detail": f"Error al leer el archivo CSV: {str(e)}"}, status=status.HTTP_400_BAD_REQUEST)
+
+        fieldnames = [f.strip().lower() for f in reader.fieldnames]
+
+        menu_week, created = MenuWeek.objects.get_or_create(
```

Ruff (0.15.12)

[warning] Unpacked variable `created` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)